pkg/bootstrap/bootstrap.go

```diff
@@ -65,8 +65,9 @@ type Options struct {
 	// enables verbose mode
 	VerboseMode bool
 
-	// enables json logger (instead of table logger)
-	JSONLogger bool
```

Comment:
The rest LGTM - but maybe update the changelog to mention that this option flag only affects the StartMulti function?